CHANGELOG.md

```diff
@@ -124,6 +124,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- **CUMULUS-1714**
```

Comment:
This seems like a breaking change. I suspect that there's some corresponding documentation that needs to be updated as well.

Reply:
In particular:

https://nasa.github.io/cumulus/docs/next/interfaces#sns-reporting-topics
https://nasa.github.io/cumulus/docs/next/data-cookbooks/ingest-notifications#sns-message-format

Reply:
Updated documentation and moved to breaking change section